sycl/include/sycl/detail/pi.h

```diff
@@ -49,7 +49,7 @@
 // NOTE that this results in a changed API for `piProgramGetBuildInfo`.
 
 #define _PI_H_VERSION_MAJOR 9
```

Comment:
The major version should also be incremented as the change to the enum technically breaks backwards compatibility. Likewise a comment should be added to the above list of changes.

Reply:
Thanks for the clarification. I have incremented the major version and added a comment.